Readme.md

```diff
@@ -3,7 +3,12 @@
 This is a starting repository if you want to run Mint in a docker container.
 
 To start the development server just clone the repository and run:
-
-```
+```sh
 docker-compose up
 ```
+
+To run the CLI via docker
+```sh
+# Replace ... with your CLI commands for mint
+docker exec -it mint-docker_mint_1 /bin/mint ...
```

Comment:
I think a better way would be:  `docker-compose run --rm mint mint start`

Reply:
Hi @gdotdesign thanks for the follow up and sorry for the late reply! Was caught up due to the COVID situation.

I would like to clarify a few questions/things and get your thoughts on these:
1. Why use docker-compose instead of a docker file directly? Is there perhaps an advantage that I am not getting?
2. Is there a reason why the binary is not baked in but instead accessed via volume mounting?
3. Specifically regarding ur suggestion about using "docker-compose run --rm mint mint start" I do not think that this is needed as the commands are already specified in the compose file right. Therefore I think the primary motive for my suggestion was to introduce a way to use this without docker compose. Nothing against it, but it just feels more inefficient and non intuitive (for deployments where I would prefer a docker image).
4. I am currently testing this on windows, and by default, it prints "⚙ Development server started on http://0.0.0.0:3000/" however this does not work on the browser nor via curl/postman. It does however works with http://localhost:3000 Is there something that I am missing here? Seems like it should work just the same?

Lastly this is slightly out of topic, but I noticed that this is always outputted, any idea why?
![image](https://user-images.githubusercontent.com/44993072/82080915-04936800-9718-11ea-8c98-ca6af2c94014.png)

Reply:
Hey @gdotdesign thanks for the follow up.

> I don't think it's better it's just different. I didn't realize you were trying to explicitly use docker and since the code is now written for docker-compose I though I would give an alternative since I thought that you are using the image which docker-compose built `mint-docker_mint_1`.

You are right, I was using the docker-compose image. Sorry for the mix up.

> It's just easier to replace without rebuilding the image.

But does this means that everytime there is a new binary, you would have to push it into this repo again? Is that not the same?

> I have nothing against it, we can have a separate docker image, something like this #1

I see, sorry didnt manage to see that issue first. Looks great, is that image updated already? Also I do think that mentioning the official image in this README would be pretty helpful. Should I add that in?

> If you are using `docker-compose up` I think it should work, however I remember that there is something about windows which is tricky but I don't know exactly what.

I see, let me try and figure this issue out and get back to you.

> I don't know either, I saw this a couple of times as well.

Okay same, I will try to see if I can do anything about it.

In the meantime should I add docs for the docker image in #1 to close up this PR? Thanks!

Reply:
> But does this means that everytime there is a new binary, you would have to push it into this repo again? Is that not the same?

Yes, it is the same.

> I see, sorry didnt manage to see that issue first. Looks great, is that image updated already? Also I do think that mentioning the official image in this README would be pretty helpful. Should I add that in?

> In the meantime should I add docs for the docker image in #1 to close up this PR? 

I didn't get to update that docker image yet, would you be interested in doing that? (I can add you as a collaborator)

Reply:
> I didn't get to update that docker image yet, would you be interested in doing that? (I can add you as a collaborator)

Sure, would love to give it a try if you don't mind. Thanks for the help anyways!

Reply:
Great! What is your Docker ID on Docker Hub?

Reply:
https://hub.docker.com/u/jaimeloeuf

Reply:
Thanks :+1: 
I've added you and gave you permission on the `mint` repository.